web/beacon-app/src/features/onboarding/components/StepButtons.tsx

```diff
@@ -0,0 +1,30 @@
+import { Trans } from '@lingui/macro';
+import { Button } from '@rotational/beacon-core';
+import React from 'react';
+
+type StepButtonsProps = {
+  isSubmitting?: boolean;
+};
+const StepButtons = ({ isSubmitting }: StepButtonsProps) => {
+  const handlePreviousClick = () => {
+    console.log('previous clicked');
+  };
+  return (
+    <div className="flex flex-row items-stretch gap-3 pt-10">
+      <Button type="submit" isLoading={isSubmitting} disabled={isSubmitting}>
```

Comment:
Should this button also be disabled until the user inserts something into the input field?

Reply:
Each step logic should handle that for now this implemented only the view